developer-guides/rest-api/channels/invite/README.md

```diff
@@ -44,6 +44,13 @@ curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
 }
 ```
 
+**NB**:
```

Comment:
@Rodriq I think we need to explain what are the permissions required to invite the user to channels, not necessarily an  `admin`, `owner` or `moderator` users can add the user, they must have some of the necessary permissions.
Here we have the respective [permissions](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/lib/server/methods/addUsersToRoom.js#L40-L46).

Reply:
Okay, I see. Let me update